buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java

```diff
@@ -215,6 +215,12 @@ private void createCorDappCPI() throws IOException, InterruptedException, CsdeEx
 //        }
 //        fileWriter.close();
 
+// todo: in the meantime, here is a simple working impl
```

Comment:
Looking at the previous code and the new one which is simple and does the job - why is it remaining as TODO ?
Why are we leaving the previous commented out code ?

Reply:
@mattbradburyr3 please confirm we can delete the old commented out code along with the TODO comment

Reply:
All of this code is being refactored into the gradle plug in. There is a todo in the gradle code to work out how to handle the error stream which references this PR. Hence we can delete the commented out code.

Reply:
done